walletclient.go

```diff
@@ -1,117 +1,151 @@
-// Package walletclient is a Go client for interacting with Spv Wallet.
 package walletclient
 
 import (
-	"github.com/bitcoin-sv/spv-wallet-go-client/transports"
+	"net/http"
+
+	"github.com/bitcoin-sv/spv-wallet/models"
 	"github.com/bitcoinschema/go-bitcoin/v2"
 	"github.com/libsv/go-bk/bec"
 	"github.com/libsv/go-bk/bip32"
 	"github.com/libsv/go-bk/wif"
 	"github.com/pkg/errors"
 )
 
-// ClientOps are used for client options
-type ClientOps func(c *WalletClient)
-
-// WalletClient is the spv wallet go client representation.
+// WalletClient is the spv wallet Go client representation.
 type WalletClient struct {
-	transports.TransportService
-	accessKey        *bec.PrivateKey
-	accessKeyString  string
-	transport        transports.TransportService
-	transportOptions []transports.ClientOps
-	xPriv            *bip32.ExtendedKey
-	xPrivString      string
-	xPub             *bip32.ExtendedKey
-	xPubString       string
+	signRequest     *bool
+	server          *string
+	accessKeyString *string
+	xPrivString     *string
+	xPubString      *string
```

Comment:
Maybe we can remove the "key" strings - because AFAIK, after initialization we use `bip32.ExtendedKey` or `bec.PrivateKey` objects.

Reply:
i removed them